tools/aa/commands/WorkspaceCommand.java

```diff
@@ -101,6 +107,22 @@ private boolean createWorkspace() {
     return true;
   }
 
+  /** Returns list from head folder with all path where file or folder to ignore is found */
+  private ImmutableList<String> getFilesOrFoldersToIgnoreInHead(String fileOrFolderToIgnore)
```

Comment:
This would not be efficient if `node_modules` has many files. We would like to ignore `node_modules` without listing att files inside.

Reply:
This method doesn't return all folders and files inside the `file_or_folder_to_ignore`. It just founds all paths with where is `file_or_folder_to_ignore`. For example, we have the following:
```
/head/repo1/a_lot_of_folders_and_files
/head/repo1/path/to/node_modules
/head/repo1/another/path/to/node_modules

/head/repo2/a_lot_of_folders_and_files
/head/repo2/some/path/to/node_modules
```
This method will return folloving list:
- `/head/repo1/path/to/node_modules`
- `/head/repo1/another/path/to/node_modules`
- `/head/repo2/some/path/to/node_modules`

Reply:
I deleted this method. The last comment isn't actual.